### PR TITLE
fix(all): add default value for jwt issuer

### PR DESCRIPTION
**Pull request by leungkinghin-ct**

### CODE REVIEW

**Review by dasanorct — APPROVED** [review]

**Comment by ddeliziact** [comment]
@leungkinghin-ct can we also change this line:

```
- CTP_CLIENT_ID: The client ID of your commercetools composable commerce user account. It is used in commercetools client to communicate with commercetools composable commerce via SDK.
```

to:
```
- CTP_CLIENT_ID: The client ID of your commercetools composable commerce user account. It is used in commercetools client to communicate with commercetools composable commerce via SDK. Expected scopes are: `manage_payments` `manage_orders` `view_sessions` `view_api_clients` `manage_checkout_payment_intents` `introspect_oauth_tokens`.
```

**Review by prateek-ct — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -158,8 +159,8 @@ Here you can see the details about various variables in configuration
 - CTP_AUTH_URL: The URL for authentication in commercetools platform. It is used to generate OAuth 2.0 token which is required in every API call to commercetools composable commerce. The default value is `https://auth.europe-west1.gcp.commercetools.com`. For details, please refer to documentation [here](https://docs.commercetools.com/tutorials/api-tutorial#authentication).
 - CTP_API_URL: The URL for commercetools composable commerce API. Default value is `https://api.europe-west1.gcp.commercetools.com`.
 - CTP_SESSION_URL: The URL for session creation in commercetools platform. Connectors relies on the session created to be able to share information between enabler and processor. The default value is `https://session.europe-west1.gcp.commercetools.com`.
-- CTP_JWKS_URL: The URL which provides JSON Web Key Set.
-- CTP_JWT_ISSUER: The issuer inside JSON Web Token which is required in JWT validation process.
+- CTP_JWKS_URL: The URL which provides JSON Web Key Set. Default value is `https://mc-api.europe-west1.gcp.commercetools.com`.
```
Comment (ddeliziact) [inline README.md:162]:
```suggestion
- CTP_JWKS_URL: The URL which provides JSON Web Key Set. Default value is `https://mc-api.europe-west1.gcp.commercetools.com/.well-known/jwks.json`. 
```

It should ge the following:

**Review by ddeliziact — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (leungkinghin-ct) [reply README.md]: https://github.com/commercetools/connect-payment-integration-adyen/pull/103/commits/879d9a079bfdcf944f6dba5b7b4741027284d615

**Comment by leungkinghin-ct** [comment]
> @leungkinghin-ct can we also change this line:
> 
> ```
> - CTP_CLIENT_ID: The client ID of your commercetools composable commerce user account. It is used in commercetools client to communicate with commercetools composable commerce via SDK.
> ```
> 
> to:
> 
> ```
> - CTP_CLIENT_ID: The client ID of your commercetools composable commerce user account. It is used in commercetools client to communicate with commercetools composable commerce via SDK. Expected scopes are: `manage_payments` `manage_orders` `view_sessions` `view_api_clients` `manage_checkout_payment_intents` `introspect_oauth_tokens`.
> ```

https://github.com/commercetools/connect-payment-integration-adyen/pull/103/commits/879d9a079bfdcf944f6dba5b7b4741027284d615

**Review by praveenkumarct — APPROVED** [review]

**Review by ddeliziact — APPROVED** [review]

**Pull request merged**